drivers/wifi/nrfwifi/Kconfig.nrfwifi

```diff
@@ -541,6 +541,15 @@ config NRF70_RSSI_STALE_TIMEOUT_MS
 
 if NETWORKING
 # Finetune defaults for certain system components used by the driver
+
+config NRF_WIFI_CTRL_HEAP_SIZE
```

Comment:
in order to leverage the newly added per-module HEAP hints, may be we should use `HEAP_MEM_POOL_ADD_SIZE_NRF70_CTRL` and ``HEAP_MEM_POOL_ADD_SIZE_NRF70_DATA`, this way there won't be any changes to the samples/applications. WDYT?

Reply:
We are creating a separate pool for each of the control path and data path, and not allocating from system heap. Since `HEAP_MEM_POOL_ADD_SIZE_*` is used just as a hint to verify the available system heap, using this name convention here may not be appropriate.

Reply:
ok, then existing `HEAP_MEM_POOL_ADD_SIZE_` have to be removed or reduced, no? And then for `NRF_WIFI_DATA_HEAP_SIZE` needs to be adjusted per-sample as per the network buffer configuration?